src/YesSql.Core/Sql/SqlBuilder.cs

```diff
@@ -79,30 +79,40 @@ public virtual void InnerJoin(string table, string onTable, string onColumn, str
             // Don't prefix if alias is used
             if (alias != onTable)
             {
-                onTable = _tablePrefix + onTable;
+                onTable = _dialect.SchemaNameQuotedPrefix() + _dialect.QuoteForTableName(onTable);
+            }
+            else
+            {
+                onTable = _dialect.QuoteForTableName(onTable);
             }
 
             if (toTable != toAlias)
+            {
+                toTable = _dialect.SchemaNameQuotedPrefix() + _dialect.QuoteForTableName(_tablePrefix + toTable);
+            }
+            else
             {
                 toTable = _tablePrefix + toTable;
             }
 
             if (!String.IsNullOrEmpty(toAlias))
             {
-                toTable = toAlias;
+                toTable = _dialect.QuoteForTableName(toAlias);
```

Comment:
Here I needed to add the `QuoteForTableName(toAlias)` because it was previously done all the time on line 113. So Alias were using `QuoteForTableName()`. Also, it is used on Alias because we also do it on line 108 but I'm not sure it is necessary to add Quotes on an Alias name in an INNER JOIN query; We could also try to remove QuoteForTableName() on lines 100 and 108 to see if tests pass.

Reply:
Seems like it's working perfectly fine without quotes.

Reply:
Well, I talked too quickly. PostgreSQL doesn't like it.